### PR TITLE
Policy server logging

### PR DESCRIPTION
**Pull request by flavio**
This PR makes the controller aware of the logging options of policy-server. This fixes https://github.com/kubewarden/kubewarden-controller/issues/43

The PR also changes the way the policy-server Deployment is updated. Prior to this PR, the Deployment definition was **never** changed by the controller.

Current limitations: the Deployment definition is updated only when a policy is added/removed. This is a know issue which is going to be tackled with another PR

### CODE REVIEW

**Comment by flavio** [comment]
@viccuad, @ereslibre I think we can go ahead, review and merge this code. By doing that @viccuad will have a solid foundation for his current work.

I'll file other issues to keep track of the other tasks to accomplish: extend the helm chart, make the controller watch also the `policy-server` ConfigMap and react on changes done to it

File: internal/pkg/admission/policy-server-deployment.go [hunk 1]
```diff
@@ -167,9 +168,20 @@ func policyServerDeploymentSettings(cfg *corev1.ConfigMap) PolicyServerDeploymen
 		settings.Image = buf
 	}
 
-	buf, found = cfg.Data[constants.PolicyServerLogLevel]
-	if found {
-		settings.EnvVars["KUBEWARDEN_LOG_LEVEL"] = buf
+	relevantEnvVarPrefixes := []string{
+		"KUBEWARDEN_", "OTEL_",
```
Comment (viccuad) [inline internal/pkg/admission/policy-server-deployment.go:160]: :100:  nice!

**Review by viccuad — APPROVED** [review]
LGTM! nice things, even if we change on reading the configmap :).

**Review by ereslibre — APPROVED** [review]
LGTM

**Comment by flavio** [comment]
Closing the PR, the new architecture makes it obsolete

**Pull request closed without merging**